src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      class="first-form  all-forms"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        class="all-images image-1"
+        src="images/Search.svg"
+        alt="magnifying glass"
+      >
+      <input
+        class="first-input all-inputs"
+        type="search"
+        name="search1"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      action="/"
+      class="second-form all-forms"
+      data-qa="small"
+      method="get"
+    >
+      <img
+        class=" all-images image-2"
+        src="images/Search.svg"
+        alt="magnifying glass"
+      >
+      <input
+        class="second-input all-inputs"
+        type="search"
+        name="search2"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
```

Comment:
lets make it work like this 
```suggestion
    <form
      action="/"
      class="form"
      method="get"
      data-qa="big"
    >
      <img
        class="image"
        src="images/Search.svg"
        alt="magnifying glass"
      >
      <input
        class="input"
        type="search"
        name="search1"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
    </form>

    <form
      action="/"
      class="form form--small"
      data-qa="small"
      method="get"
    >
      <img
        class=" image image--small"
        src="images/Search.svg"
        alt="magnifying glass"
      >
      <input
        class="input input--small"
        type="search"
        name="search2"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```
lets make it work like this